selfdrive/car/hyundai/interface.py

```diff
@@ -153,6 +153,14 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
       if candidate == CAR.KIA_NIRO_HEV:
         ret.minSteerSpeed = 32 * CV.MPH_TO_MS
+    elif candidate == CAR.KIA_NIRO_HEV_2021:
```

Comment:
same parameters as CAR.KIA_NIRO_HEV, just add it to the list above

Reply:
Done!